client/reader/list-gap/index.jsx

```diff
@@ -13,9 +13,10 @@ var FeedStreamStoreActions = require( 'lib/feed-stream-store/actions' ),
 var Gap = React.createClass( {
 
 	propTypes: {
-		gap: React.PropTypes.object.isRequired,
+		gap: React.PropTypes.object,
+		postKey: React.PropTypes.object,
```

Comment:
according to `client/lib/feed-stream-store/feed-stream.js`, a gap _is a_ type of postKey